pkg/graveler/graveler_v2_test.go

```diff
@@ -388,3 +388,51 @@ func TestGravelerRevert(t *testing.T) {
 	})
 
 }
+
+func TestGravelerCommit(t *testing.T) {
+	ctx := context.Background()
+
+	t.Run("commit with sealed tokens", func(t *testing.T) {
```

Comment:
Looks great - let's add more scenarios like commits which is being retried because the tokens were changed, empty commit, etc.

Reply:
Most of the cases are tested in other places, I wanted to add only tests that related to the new functionality.
I couldn't find a way to simulate changing the tokens while updating the branch.

Reply:
Look at "revert dirty branch after token update" for example.
You can use gomock to return different results on the first/second/third/.../nth request to the same mocked entity

Reply:
discussed f2f, already done